src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -262,10 +262,10 @@ public static Exception LogExceptionMessage(Exception exception)
         public static Exception LogExceptionMessage(EventLevel eventLevel, Exception exception)
         {
             if (exception == null)
-                return null;
+                return null!;
 
             if (IdentityModelEventSource.Logger.IsEnabled(eventLevel, EventKeywords.All))
-                IdentityModelEventSource.Logger.Write(eventLevel, exception.InnerException, exception.Message);
+                IdentityModelEventSource.Logger.Write(eventLevel, exception.InnerException!, exception.Message);
```

Comment:
how do we know innerexception is not null here?

Reply:
+1, also, I don't think this is needed.

Reply:
Same as the next comment

Reply:
Made arg nullable